snc.sh

```diff
@@ -47,13 +47,13 @@ fi
 mkdir -p openshift-clients/linux openshift-clients/mac openshift-clients/windows
 if [[ ${OKD_VERSION} != "none" ]]
 then
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-linux-${OPENSHIFT_RELEASE_VERSION}.tar.gz" | tar -zx -C openshift-clients/linux oc
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-mac-${OPENSHIFT_RELEASE_VERSION}.tar.gz" | tar -zx -C openshift-clients/mac oc
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-windows-${OPENSHIFT_RELEASE_VERSION}.zip" > openshift-clients/windows/oc.zip
+    curl -L "${MIRROR}/openshift-client-linux-${OPENSHIFT_RELEASE_VERSION}.tar.gz" | tar -zx -C openshift-clients/linux oc
```

Comment:
This should be `curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-linux.tar.gz"` , and we can remove this `OKD` check here because now we have the tarball without release version stuff on ocp side.

Reply:
I set OPENSHIFT_VERSION directly to pre-release. The change is smaller with that.